src/test/java/org/mabrarov/exceptionsafety/FactoryMethodTest.java

```diff
@@ -32,193 +33,212 @@
 
 public class FactoryMethodTest {
 
-  @Rule
```

Comment:
Please consider following existing formatting. Use 2 spaces for indentation and 4 spaces for continuation. This way the diff will become smaller and this pull request will become easier to read and to understand.